lib/block/test.go

```diff
@@ -2,13 +2,72 @@ package block
 
 import (
 	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/consensus/round"
+	"boscoin.io/sebak/lib/network"
+	"boscoin.io/sebak/lib/node"
+	"boscoin.io/sebak/lib/transaction"
 	"github.com/stellar/go/keypair"
 )
 
+var networkID []byte = []byte("sebak-test-network")
+
 func TestMakeBlockAccount() *BlockAccount {
 	kp, _ := keypair.Random()
 	address := kp.Address()
 	balance := common.Amount(2000)
 
 	return NewBlockAccount(address, balance)
 }
+
+var (
+	kp           *keypair.Full
+	account      *BlockAccount
+	genesisBlock Block
+)
+
+func init() {
+	kp, _ = keypair.Random()
+}
+
+func createNetMemoryNetwork() (*network.MemoryNetwork, *node.LocalNode) {
```

Comment:
This was removed in https://github.com/bosnet/sebak/pull/351 and shouldn't have been added back

Reply:
thx mat!
let's do this kind of missed things on the other PR.
I will make it.